glusterd2/events/events.go

```diff
@@ -0,0 +1,64 @@
+package events
+
+import (
+	"strings"
+
+	"github.com/pborman/uuid"
+)
+
+// Event represents an event in GD2
+type Event struct {
+	// ID is a unique event ID
+	ID uuid.UUID
+	// Name is the the name of the event
+	Name string
+	// Data is any additional data attached to the event.
+	Data map[string]string
+	// global should be set to true to broadcast event to the full GD2 cluster.
```

Comment:
We may need a timestamp too. And this timestamp can be part of the etcd key. We can then do a ranged gets from timestamp1 to timestamp2.